node_package/src/ReactOnRails.ts

```diff
@@ -30,6 +30,16 @@ if (ctx === undefined) {
   throw new Error("The context (usually Window or NodeJS's Global) is undefined.");
 }
 
+if (ctx.ReactOnRails !== undefined) {
+  throw new Error(`
+    The ReactOnRails value exists in the ${ctx} scope, it may not be safe to overwrite it.
+    
+    This could be caused by setting Webpack's optimization.runtimeChunk to "true" or "multiple" - please check your Webpack configuration.
```

Comment:
This could be caused by setting Webpack's optimization.runtimeChunk to "true" or "multiple," rather than "single." Check your Webpack configuration.

Reply:
Done in cbde473a203dbc230e4431a1df69c28ddc2bfe26.